app/export.go

```diff
@@ -62,6 +65,31 @@ var exportablePreferences = map[ComparablePreference]string{{
 }: "EmailInterval",
 }
 
+func (a *App) GetId() string {
+	return a.Id
+}
+func (a *App) SetId(id string) {
+	a.Id = id
+}
+
+type LogFormat struct {
```

Comment:
move this to a file which have other type declarations we just cannot define a type anywhere in our code

Reply:
Okk